web-console/src/views/workbench-view/execution-summary-panel/execution-summary-panel.tsx

```diff
@@ -97,10 +98,14 @@ export const ExecutionSummaryPanel = React.memo(function ExecutionSummaryPanel(
           className="download-button"
           content={
             <Menu>
-              <MenuDivider title="Download results as..." />
+              <MenuDivider title="Save Query Results:" />
```

Comment:
The capitalization here (and everywhere) should be sentance case so `Save query results`.

What do you think of, instead of changing this MenuDivider add another one.

So you will have

```
Download results as...
  CSV
  TSV
  JSON (new line delimited)
Copy to clipboard as...
  CSV
  TSV
  JSON (new line delimited)
```

That way people can copy all the formats

Reply:
Good suggestion, I will refactor accordingly